### DISCUSSION TITLE
360 t7 主分支 custom U-Boot layout 为什么删了

### DISCUSSION THREAD

**Discussion by liyiranlab** [immortalwrt/immortalwrt#1154] [General]
360 t7 主分支 custom U-Boot layout 为什么删了，昨天（202402012）编译360 t7还有immortalwrt-mediatek-filogic-qihoo_360t7-ubootmod-squashfs-sysupgrade.bin的大分区固件，今天编译没有了。主要是听说官方的uboot不支持联发科的rom坏块管理，所以不想使用官方的uboot。希望天灵大佬恢复对hanwckf大佬的uboot支持，谢谢。

**Comment by xlighting2017** [comment]
hanwckf大佬的uboot， 在 360t7 上， 有/支持联发科的rom坏块管理 么？

**Reply by liyiranlab** [reply]
应该有吧？

**Reply by liyiranlab** [reply]
https://www.right.com.cn/forum/thread-8292288-1-1.html

**Reply by liyiranlab** [reply]
https://cmi.hanwckf.top/p/mt798x-uboot-usage/

**Reply by liyiranlab** [reply]
由于OpenWrt主线的ubootmod固件未开启nmbm支持，因此不支持任何ubootmod分区布局的主线固件（包括OpenWrt, X-Wrt等带有ubootmod字样的固件），请使用OpenWrt主线提供的uboot启动ubootmod固件

**Comment by 1715173329** [comment]
NMBM 本身就是个很烂的想法，经过评估维护成本已经大于保留这种 layout 的意义，毕竟又没见几个人来提 PR。

**Reply by liyiranlab** [reply]
之前我的t7一直在用天灵大佬的固件，直到大佬删除custom U-Boot layout ，我才发现天灵大佬的T7固件没有开启NMBM，但是我觉得天灵大佬的主线无线比h大的无线要好用一点，可能是因为我组网的R3G，AX6也用的天灵大佬的固件，现在没办法只能使用L大的LEDE和天灵大佬的R3G，AX6的固件配合。效果要比h大的闭源好点，h大的只有kv，没有看见r。

**Reply by liyiranlab** [reply]
L大的T7固件也开启了NMBM的。

**Reply by liyiranlab** [reply]
我翻了一下网上帖子，感觉好像OpenWrt官方uboot没有web刷机页面，没有NMBM，一旦无法加入op，刷固件就非常不方便了呀！

**Reply by liyiranlab** [reply]
就用户量来说，可能大部分网友都刷的h的uboot吧。

**Reply by star-jj** [reply]
目前比较头疼的是后期的t7改了闪存芯片的配置，目前能支持的就h大的uboot，后续没有了就很头疼

**Reply by Dichgrem** [reply]
现在24.10.0的immortalwrt不支持ubootmod,360T7最高就只能刷23.05了

**Reply by 1715173329** [reply]
> 目前比较头疼的是后期的t7改了闪存芯片的配置，目前能支持的就h大的uboot，后续没有了就很头疼

主线去年就完全支持了，不存在“就”。24.10.0 完全支持；23.05.4 发布较早只有内核支持，可以用 24.10 的 uboot 引导。
